keras_rs/src/layers/retrieval/sampling_probability_correction.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+import keras
+from keras import ops
+
+from keras_rs.src import types
+from keras_rs.src.api_export import keras_rs_export
+
+
+@keras_rs_export("keras_rs.layers.SamplingProbablityCorrection")
+class SamplingProbablityCorrection(keras.layers.Layer):
+    """Sampling probability correction.
+
+    Args:
+        epsilon: Small float added to sampling probability to avoid taking the
+            log of zero. Defaults to 1e-3.
+        **kwargs: Args to pass to the base class.
```

Comment:
Is it standard practice in Keras to specify kwargs in the doc-string? Haven't seen it elsewhere.

Reply:
Some internal tools flags the docstring as being incorrect if it's not in the args. So I took the habit of always adding it even if it typically does not provide useful information.